### ISSUE TITLE
Invalid player data

### ISSUE THREAD

**Issue by Yurzi** [TartaricAcid/ysm#76]
## Environment:
Java: openjdk 17.0.10
Fabric: fabric-server-mc.1.19.2-loader.0.15.7-launcher.1.0.0.jar
Fabric-API: fabric-api-0.77.0+1.19.2.jar
YSM: yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar

## 描述：

无法进入装有此版本ysm的服务器，但单机和联机使用正常。

## Error Log:
进入从单机迁移到服务器的存档
```log
[14:37:59] [User Authenticator #2/INFO]: UUID of player Yurzi is xxxxx-xxxxx-xxxx-xxxxxx-xxxxxxx
[14:38:00] [Server thread/ERROR]: Couldn't place player in world
net.minecraft.class_148: Loading entity NBT
	at net.minecraft.class_1297.method_5651(class_1297.java:1815) ~[server-intermediary.jar:?]
	at net.minecraft.class_29.method_261(class_29.java:56) ~[server-intermediary.jar:?]
	at net.minecraft.class_3324.method_14600(class_3324.java:360) ~[server-intermediary.jar:?]
	at net.minecraft.class_3324.method_14570(class_3324.java:151) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_33800(class_3248.java:151) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_14384(class_3248.java:138) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.redirect$zge000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_18784(class_3248.java:73) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10754(class_2535.java:248) ~[server-intermediary.jar:?]
	at net.minecraft.class_3242.method_14357(class_3242.java:183) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:886) ~[server-intermediary.jar:?]
	at net.minecraft.class_3176.method_3813(class_3176.java:289) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.elfmcys.yesstevemodel.OoOoOOO00OoOo00oooOoo00
	at net.minecraft.class_1657.OooOo0oOoOO0o00OO000oO00(class_1657.java:4869) ~[server-intermediary.jar:?]
	at net.minecraft.class_1657.handler$bba000$yes_steve_model$OooOo0oOoOO0o00OO000oO00(class_1657.java:4876) ~[server-intermediary.jar:?]
	at net.minecraft.class_1657.method_5749(class_1657.java:867) ~[server-intermediary.jar:?]
	at net.minecraft.class_3222.method_5749(class_3222.java:316) ~[server-intermediary.jar:?]
	at net.minecraft.class_1297.method_5651(class_1297.java:1806) ~[server-intermediary.jar:?]
	... 15 more
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchFieldError: OooOo0oOoOO0o00OO000oO00 [in thread "Server thread"]
	at com.elfmcys.yesstevemodel.OoOoOOO00OoOo00oooOoo00.<clinit>(Unknown Source) ~[yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar:?]
	at net.minecraft.class_1657.OooOo0oOoOO0o00OO000oO00(class_1657.java:4869) ~[server-intermediary.jar:?]
	at net.minecraft.class_1657.handler$bba000$yes_steve_model$OooOo0oOoOO0o00OO000oO00(class_1657.java:4876) ~[server-intermediary.jar:?]
	at net.minecraft.class_1657.method_5749(class_1657.java:867) ~[server-intermediary.jar:?]
	at net.minecraft.class_3222.method_5749(class_3222.java:316) ~[server-intermediary.jar:?]
	at net.minecraft.class_1297.method_5651(class_1297.java:1806) ~[server-intermediary.jar:?]
	... 15 more
[14:38:00] [Server thread/INFO]: com.mojang.authlib.GameProfile@2d848ba8[id=3da632d1-8552-447f-851d-016c223e97b8,name=Yurzi,properties={textures=[com.mojang.authlib.properties.Property@339c4a34], uploadableTextures=[com.mojang.authlib.properties.Property@3d5ba237]},legacy=false] (/125.113.167.125:48802) lost connection: Invalid player data
[14:38:00] [Server thread/WARN]: handleDisconnection() called twice
stop
```

进入服务器原生的存档
```log
[14:50:48] [User Authenticator #1/INFO]: UUID of player Yurzi is xxxxxxx-xxxxxx-xxxxx-xxxxxxx-xxxxxxx
[14:50:49] [Server thread/INFO]: Yurzi[/xxxxxxxxxx:xxxx] logged in with entity id 344 at (19.5, 71.0, -50.5)
[14:50:49] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoSuchFieldError: OooOo0oOoOO0o00OO000oO00
	at com.elfmcys.yesstevemodel.OoOoOOO00OoOo00oooOoo00.<clinit>(Unknown Source) ~[yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar:?]
	at net.minecraft.class_1657.OooOo0oOoOO0o00OO000oO00(class_1657.java:4869) ~[server-intermediary.jar:?]
	at com.elfmcys.yesstevemodel.OoOOOoO0ooo0oO000O0o0O.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar:?]]: Starting minecraft server version 1.19.2
	at com.elfmcys.yesstevemodel.ooOoO0OOooO00oOo00oooOoo.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar:?]]: Generating keypair
	at com.elfmcys.yesstevemodel.Ooo00OOoOO0oOOoO000o0oo0.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.19.2-fabric-release-1.1.5-hotfix-2.jar:?]]: Preparing level "world"
	at net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents.lambda$static$2(ServerPlayConnectionEvents.java:48) ~[fabric-networking-api-v1-1.2.12+def3f86d90-79c6c1b478adf57f.jar:?]
	at net.fabricmc.fabric.impl.networking.server.ServerPlayNetworkAddon.onClientReady(ServerPlayNetworkAddon.java:65) ~[fabric-networking-api-v1-1.2.12+def3f86d90-79c6c1b478adf57f.jar:?]
	at net.minecraft.class_3324.handler$zgd000$fabric-networking-api-v1$handlePlayerConnection(class_3324.java:4013) ~[server-intermediary.jar:?]
	at net.minecraft.class_3324.method_14570(class_3324.java:200) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_33800(class_3248.java:151) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_14384(class_3248.java:138) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.redirect$zge000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563) ~[server-intermediary.jar:?]
	at net.minecraft.class_3248.method_18784(class_3248.java:73) ~[server-intermediary.jar:?]
	at net.minecraft.class_2535.method_10754(class_2535.java:248) ~[server-intermediary.jar:?]
	at net.minecraft.class_3242.method_14357(class_3242.java:183) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:886) ~[server-intermediary.jar:?]
	at net.minecraft.class_3176.method_3813(class_3176.java:289) ~[server-intermediary.jarver thread/INFO]: Time elapsed: 11441 ms
:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]
[14:50:49] [Server thread/ERROR]: This crash report has been saved to: /root/minecraft/./crash-reports/crash-2024-03-03_14.50.49-server.txt
```

**Comment by Mrpressed** [comment]
Had this same issue, when I went onto a new world it just crash immediately.

**Comment by passworked** [comment]
same
```[17:49:38] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoSuchFieldError: OooOo0oOoOO0o00OO000oO00
        at com.elfmcys.yesstevemodel.OoOoOOO00OoOo00oooOoo00.<clinit>(Unknown Source) ~[yesstevemodel-1.20-fabric-release-1.1.5-hotfix-2.jar:?]
        at net.minecraft.class_1657.OooOo0oOoOO0o00OO000oO00(class_1657.java:6388) ~[server-intermediary.jar:?]
        at com.elfmcys.yesstevemodel.OoOOOoO0ooo0oO000O0o0O.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.20-fabric-release-1.1.5-hotfix-2.jar:?]
        at com.elfmcys.yesstevemodel.ooOoO0OOooO00oOo00oooOoo.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.20-fabric-release-1.1.5-hotfix-2.jar:?]
        at com.elfmcys.yesstevemodel.Ooo00OOoOO0oOOoO000o0oo0.OooOo0oOoOO0o00OO000oO00(Unknown Source) ~[yesstevemodel-1.20-fabric-release-1.1.5-hotfix-2.jar:?]
        at net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents.lambda$static$2(ServerPlayConnectionEvents.java:48) ~[fabric-networking-api-v1-1.3.11+1802ada577-a65c3ef3908d32fd.jar:?]
        at net.fabricmc.fabric.impl.networking.server.ServerPlayNetworkAddon.onClientReady(ServerPlayNetworkAddon.java:66) ~[fabric-networking-api-v1-1.3.11+1802ada577-a65c3ef3908d32fd.jar:?]
        at net.minecraft.class_3324.handler$zjm000$fabric-networking-api-v1$handlePlayerConnection(class_3324.java:3018) ~[server-intermediary.jar:?]
        at net.minecraft.class_3324.method_14570(class_3324.java:215) ~[server-intermediary.jar:?]
        at net.minecraft.class_3248.method_33800(class_3248.java:131) ~[server-intermediary.jar:?]
        at net.minecraft.class_3248.method_14384(class_3248.java:118) ~[server-intermediary.jar:?]
        at net.minecraft.class_3248.redirect$zjn000$fabric-networking-api-v1$handlePlayerJoin(class_3248.java:563) ~[server-intermediary.jar:?]
        at net.minecraft.class_3248.method_18784(class_3248.java:68) ~[server-intermediary.jar:?]
        at net.minecraft.class_2535.method_10754(class_2535.java:259) ~[server-intermediary.jar:?]
        at net.minecraft.class_3242.method_14357(class_3242.java:172) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:908) ~[server-intermediary.jar:?]
        at net.minecraft.class_3176.method_3813(class_3176.java:283) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:824) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:671) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[server-intermediary.jar:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[17:49:38] [Server thread/ERROR]: This crash report has been saved to: /workspace/./crash-reports/crash-2024-06-11_17.49.38-server.txt
[17:49:38] [Server thread/INFO]: Stopping server
[17:49:38] [Server thread/INFO]: Saving players
[17:49:38] [Server thread/INFO]: Saving worlds
[17:49:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:overworld
[17:49:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_nether
[17:49:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_end
[17:49:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (world): All chunks are saved
[17:49:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[17:49:41] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[17:49:41] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved`se``
server crashed while player joining.

**Comment by TartaricAcid** [comment]
Fix in 2.2.1
